pyanaconda/core/constants.py

```diff
@@ -84,7 +84,7 @@
 DRACUT_SHUTDOWN_EJECT = "/run/initramfs/usr/lib/dracut/hooks/shutdown/99anaconda-eject.sh"
 
 # VNC questions
-USEVNC = N_("Start VNC")
+USERDP = N_("Use remote graphical mode")
```

Comment:
I think we should mention here RDP with capital letters ``Use Remote Desktop Protocol graphical mode``. My point is that it should clarify what user should use to do the connection to the installation.